source/blender/gpu/intern/gpu_shader.c

```diff
@@ -535,7 +535,7 @@ void GPU_shader_free(GPUShader *shader)
 
 int GPU_shader_get_uniform(GPUShader *shader, const char *name)
 {
-	return glGetUniformLocation(shader->program, name);
+	return shader? glGetUniformLocation(shader->program, name) : -1;
```

Comment:
It's a temp fix for filter.setTexture(0, bindcode, name)... I already had this error but I though it was fixed. The shader can apparently be NULL so when we call GetUniformLocation in setTexture we have a crash. Iirc you had linked glsl shader at the first frame to fix this bug but maybe it has been refactored and the fix disappeared...